pkg/manager-nvme/nvme_cli.go

```diff
@@ -278,6 +278,51 @@ func (d *cliDevice) DeleteNamespace(namespaceId nvme.NamespaceIdentifier) error
 	return nil
 }
 
+func (d *cliDevice) FormatNamespace(namespaceID nvme.NamespaceIdentifier) error {
+	// Example Command
+	//    # nvme format --force --namespace-id=1 /dev/nvme2
+	//    format: Success, formatted nsid:1
+
+	rsp, err := d.run(fmt.Sprintf("format %s --force --namespace-id=%d", d.dev(), namespaceID))
+	if err != nil {
+		return err
+	}
+
+	regex := regexp.MustCompile(`^format: Success, formatting nsid:(?P<NSID>\d+)\n`)
```

Comment:
Should do this outside of the function so it only compiles once.

Reply:
And I just noticed this is consistent with the other functions, so you can ignore this.